firefox_puppeteer/tests/test_toolbars.py

```diff
@@ -223,8 +223,7 @@ def tearDown(self):
 
     @skip_under_xvfb
     def test_elements(self):
-        self.locationbar.identity_box.click()
-        Wait(self.marionette).until(lambda _: self.identity_popup.is_open)
+        self.open()
```

Comment:
the `open` method is part of the `IdentityPopup` class but not of the current test function.